testing/jormungandr-scenario-tests/src/node.rs

```diff
@@ -760,21 +760,14 @@ impl<'a, R: RngCore, N> SpawnBuilder<'a, R, N> {
         }
     }
 
-    fn set_log_level(&mut self, log_file: &Path) {
+    fn set_log_level(&mut self, _log_file: &Path) {
         let format = "plain";
         let level = self.context.log_level();
-        self.node_settings.config.log = Some(Log(vec![
-            LogEntry {
-                format: format.to_string(),
-                level: level.to_string(),
-                output: LogOutput::Stderr,
-            },
-            LogEntry {
-                format: format.to_string(),
-                level,
-                output: LogOutput::File(log_file.to_path_buf()),
-            },
-        ]));
+        self.node_settings.config.log = Some(Log(LogEntry {
+            format: format.to_string(),
```

Comment:
This will break because we are only able to read from json encoded logs, while plain format is used here